### DISCUSSION TITLE
Unable to install Pro (got rate limited)

### DISCUSSION THREAD

**Discussion by rahman-nero** [Laravel-Backpack/community-forum#856] [Q&A (Help)] [answered]
Around 5 days ago I installed backpack for 3-5 times because I was changing dependencies in my project (I was upgrading from laravel 8 to 10). Somehow composer didn't take from cache and each time it downloaded it again so I got rate limited. Today I tried install it again but got the same error

```
https://repo.backpackforlaravel.com could not be fully loaded (The "https://repo.backpackforlaravel.com/packages.json" file could not be downloaded (HTTP/2 429 )), package information was loaded from the lo....
```

After 5 days I got the same error, so when will my ban be lifted?. My IP: 94.230.128.60 (I am using proxy server)

**Comment by tabacitu** [comment, marked as answer]
Hey there,

We've seen A LOT of requests from that IP, so yes our firewall has blocked it. Let's try this - please go to https://repo.backpackforlaravel.com - you should see a JS/captcha challenge you can manually do. AFAIK, after that, CloudFlare should let you in.

If that doesn't fix it for you, please email me personally - cristian.tabacitu@backpackforlaravel.com - I have more questions but would rather ask them privately.

Cheers!

**Comment by jcastroa87** [comment]
Hello @rahman-nero, since there is already an answer here, I will close the case, but if you need to reopen it, feel free or create a new one if necessary.

Cheers.
